build/test-requirements.txt

```diff
@@ -8,3 +8,4 @@ pillow
 pytest-benchmark
 pytest-xdist
 wheel
+torch
```

Comment:
I'm not sure we should start adding torch as a test dep.

Reply:
removed.